frontend/javascripts/oxalis/view/settings/setting_input_views.js

```diff
@@ -24,17 +24,21 @@ export class NumberSliderSetting extends React.PureComponent<NumberSliderSetting
   };
 
   _onChange = (_value: number) => {
-    if (this.props.min <= _value && _value <= this.props.max) {
+    if (this.isValueValid(_value)) {
       this.props.onChange(_value);
     }
   };
 
+  isValueValid = (_value: number) =>
+    _.isNumber(_value) && _value >= this.props.min && _value <= this.props.max;
+
```

Comment:
Nice that you extracted the check into an own method to reduce code duplication :+1: